apps/src/aichat/types.ts

```diff
@@ -26,12 +26,18 @@ export enum Role {
 
 export enum Status {
   ERROR = 'error',
+  PROFANITY = 'profanity',
+  PERSONAL = 'personal',
   INAPPROPRIATE = 'inappropriate',
   OK = 'ok',
-  PERSONAL = 'personal',
   UNKNOWN = 'unknown',
+  EMAIL = 'email',
+  ADDRESS = 'address',
+  PHONE = 'phone',
```

Comment:
super nit: "phone number"?

Reply:
I need these to match the [strings that the share filter returns](https://github.com/code-dot-org/code-dot-org/blob/cf3667238e3aeff62f1f3d8e10c6cac21a75f49a/lib/cdo/share_filtering.rb#L41).